rest/src/db/CatapultDb.js

```diff
@@ -423,6 +432,65 @@ class CatapultDb {
 			})));
 	}
 
+	addBlockMetaToTransactionList(list) {
+		return this.addBlockMetaToEntityList(list, ['timestamp', 'feeMultiplier'],
+			item => item.meta.height);
```

Comment:
if this doesn't fit on one line, we usually drop all params on their own lines

Reply:
one liner now